k8s/namespaces/rd/rd-commondata-api/perftest.yaml

```diff
@@ -7,4 +7,4 @@ spec:
     java:
       environment:
         DELETE_ORG: true
-        CRD_S2S_AUTHORISED_SERVICES: rd_commondata_api,xui_webapp
+        CRD_S2S_AUTHORISED_SERVICES: rd_commondata_api,xui_webapp,sptribs_case_api
```

Comment:
Hi @vasthav-patta is it not required in ITHC ?

Reply:
as per ticket we need to whitelist in below env
AAT, Demo,Perftest

Reply:
ok Thank you @vasthav-patta for the clarification :)